scripts/registry_parser.py

```diff
@@ -0,0 +1,147 @@
+"""
+Script to process the registry and transform the mapping sets into Turtle to
+upload on OxO2
+"""
+
+import argparse
+import uuid
+from typing import List, Tuple
+
+import yaml
+from pyld.jsonld import expand
+from rdflib import Graph
+from sssom.parsers import parse_sssom_table
+from sssom.writers import to_json
+from sssom_schema import SSSOM, MappingRegistry, MappingSetReference
+
+
+def registry_parser(config: str) -> MappingRegistry:
+    """ Parse registry and return MappingRegistry """
+    with open(file=config, mode="r", encoding="utf-8") as f:
+        data = yaml.safe_load(f)
+
+    map_set_refs = (
+        MappingSetReference(
```

Comment:
since this is a data class, should this not work just as well: `MappingSetReference(mapping)`?

Reply:
No, because there're fields not in the MappingSetReference. For example `mapping_set_confidence` and `others`. 

```bash
ValueError:  Unknown argument: mapping_set_confidence = '0.99'
Unknown argument: other = {'local_id': 'mondo'}
```

Reply:
Ah ok!